### ISSUE TITLE
[Beta] GPT translation error

### ISSUE THREAD

**Issue by rcochrane55** [DraqueT/PolyGlot#1316]
I'm trying out the latest beta and the GPT translation tool. I set up an API key through OpenAI and put it into the field in settings, set up a grammar section as text only (removed images of tables) and then tried having GPT translate a simple noun phrase. However, when I click translate, I receive the following error message:
![data body problem](https://github.com/DraqueT/PolyGlot/assets/67019264/3c7c8e26-c505-4b79-9fef-623235c380a2)

As far as I can tell, I've done everything right to set up the tool, and I don't really understand what the error means. Has this come up before?

**Comment by DraqueT** [comment]
Thank you for bringing this to my attention! I will check this out on my Windows test environment as soon as I get the chance. I am not able to reproduce this on my current Mac build and am wondering if there's some platform specific quirk going on.

Which grammar sections do you have selected there? It is possible that there is something included in one that is not being encapsulated correctly when I package it up for GPT.

The error itself means that the message that is being built and sent to the GPT servers is somehow malformed.

**Comment by rcochrane55** [comment]
The only grammar section selected here is "Nouns and Pronouns" and I have set that section up as text-only. Normally, I use screenshots of tables I made in google docs in the grammar section, but GPT can't accept image input, so I took the information in those tables and formatted them as text. I don't see anything in that section that GPT shouldn't be able to process. I can send over the file again too if that would be helpful. My grammar section needs a pretty extensive rewrite in any case for clarity and concision, so I decided to start small in testing out the GPT translation and just set up one section for it initially to try out very simple translations.
